engine/src/main/resources/org/camunda/bpm/engine/impl/mapping/entity/Task.xml

```diff
@@ -422,6 +422,9 @@
             <if test="query.keyLike != null">
               ${queryType} RES.TASK_DEF_KEY_ like #{query.keyLike} ESCAPE ${escapeChar}
             </if>
+            <if test="query.keyNotLike != null">
+              ${queryType} RES.TASK_DEF_KEY_ like #{query.keyNotLike} ESCAPE ${escapeChar}
```

Comment:
This should be `not like` if I'm not mistaken.

Reply:
yes .
The sql should be not like..